modules/seqexec/engine/src/test/scala/seqexec/engine/packageSpec.scala

```diff
@@ -218,43 +219,42 @@ class packageSpec extends FlatSpec with NonImplicitAssertions {
 
   "engine" should "keep processing input messages regardless of how long Actions take" in {
     val result = (for {
-      q <- Stream.eval(async.boundedQueue[IO, executionEngine.EventType](10))
+      q           <- Stream.eval(async.boundedQueue[IO, executionEngine.EventType](1))
       startedFlag <- Stream.eval(Semaphore.apply[IO](0))
-      finishFlag <- Stream.eval(Semaphore.apply[IO](0))
-    } yield {
-      val qs = Engine.State[Unit]((),
-        Map((seqId, Sequence.State.init(Sequence(
-          id = "First",
-          metadata = SequenceMetadata(GmosS, None, ""),
-          steps = List(
-            Step.init(
-              id = 1,
-              fileId = None,
-              config = config,
-              resources = Set(GmosS),
-              executions = List(
-                List(fromIO(ActionType.Configure(TCS),
-                  IO.apply {
-                    startedFlag.decrement
-                    finishFlag.increment
-                    Result.OK(Result.Configured(TCS))
-                  }))
```

Comment:
This yield was producing a Stream[IO, IO[List[Unit]], instead of a Stream[IO, List[Unit]]. The code inside was never evaluated.

Reply:
👍